pkg/api/serialization_test.go

```diff
@@ -44,21 +31,14 @@ import (
 	// install all APIs
 	_ "github.com/openshift/origin/pkg/api/install"
 	_ "github.com/openshift/origin/pkg/quota/api/install"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	_ "k8s.io/kubernetes/pkg/api/install"
+	"k8s.io/kubernetes/pkg/apis/componentconfig"
 )
 
-var codecsToTest = []func(version schema.GroupVersion, item runtime.Object) (runtime.Codec, error){
```

Comment:
@smarterclayton PTAL at this commit... much of this was stuff you set up and I'm not sure the intent of some of it